docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,61 @@
+# Kubebuilder v1 vs v2
+
+There are a number of changes in
```

Comment:
Lang: this feels a bit awkward or redundant.  I think this either needs to go away, or be replaced by a different intro.